src/components/FAQ-page/faq-style.css

```diff
@@ -0,0 +1,15 @@
+h1{
```

Comment:
This is not how we add CSS in React.js
Checkout the other components for reference

Reply:
I will try to remove faq-style file once question file will get executed